server/src/internalClusterTest/java/org/opensearch/snapshots/CloneSnapshotIT.java

```diff
@@ -177,25 +177,57 @@ public void testCloneShallowSnapshotIndex() throws Exception {
         createIndex(remoteStoreEnabledIndexName, remoteStoreEnabledIndexSettings);
         indexRandomDocs(remoteStoreEnabledIndexName, randomIntBetween(5, 10));
 
+        assertBusy(
```

Comment:
I don't think `assertBusy()` will do anything here since you're not asserting anything. I believe it will only retry AssertionErrors, and your code block would throw a different exception upon timeout or failure. I think you don't actually need assertBusy() here since the client call will block until that status is reached. I'd also consider looking at whether one of the `ensureYellow()` helper methods in the base class can be used here.